app/services/translation/gemma_client.py

```
@@ -0,0 +1,40 @@
+            f"Adhere strictly to the following rules:\n"
+            f"1. Use standard Devanagari script for all Hindi text.\n"
+            f"2. Ensure the grammar and tone are appropriate for a formal, technical context.\n"
+            f"3. CRITICAL: The text contains special protected tokens formatted as [0001], [0002], etc. You MUST NOT translate, modify, or remove these tokens. Leave the brackets and numbers exactly as they appear in the original text.\n"
+            f"4. Do NOT provide any introductory or concluding remarks. Return ONLY the translated text.\n\n"
+            f"English text to translate:\n{text}"
+        )
+
+        try:
+            response = ollama.generate(model=self.model_name, prompt=prompt)
+            return response.get('response', text).strip()
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)